src/systems/a320_systems/src/lib.rs

```diff
@@ -115,6 +115,8 @@ impl Aircraft for A320 {
             &self.hydraulic_overhead,
             &self.engine_fire_overhead,
             &self.landing_gear,
+            &self.emergency_electrical_overhead,
+            self.electrical.in_emergency_elec(),
```

Comment:
**thought**: We might consider adding a trait for this too, such that we don't pass a bool around but something more meaningful. What do you think?

Reply:
Yup, done... well I think if my computer let me save a file :D